css/media.css

```diff
@@ -0,0 +1,388 @@
+@media (max-width: 1220px) {
+   .welcome-page {
+     width: 100%;
+     margin: 0 auto;
+   }
+ }
+
+ @media (max-width: 1036px) {
+   .header__menu_link {
+     font-size: 12px;
+     line-height: 15px;
+   }
+
+   .link__item {
+     font-size: 12px;
+     line-height: 15px;
+   }
+ }
+
+ @media (max-width: 856px) {
+   .header__menu_link {
+     display: none;
+   }
+
+   .welcome-page__speed-car {
+     width: 80%;
+   }
+
+   .welcome-page__speed-sec {
+     font-size: 28px;
+     line-height: 33px;
+   }
+
+   .link-none {
+     display: none;
+   }
+
+   .welcome-page__block {
+     width: 50%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)